packages/puppeteer-core/src/bidi/core/Browser.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * @license
```

Comment:
Can you create an issue to create an EsLint rule to check/add this.

Reply:
It'd be faster if you do this. I'm not sure what you mean.

Reply:
Opened #11701.